tests/Test.hs

```diff
@@ -0,0 +1,255 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+{-# OPTIONS_GHC -fno-warn-deprecations #-}
+{-# OPTIONS_GHC -fno-warn-unused-imports #-}
+
+module Main (main) where
+
+import Control.Applicative
+import Control.Concurrent
+import qualified Control.Exception as E
+import Control.Monad
+import Data.List (sort)
+import System.Exit
+import System.IO
+import System.Posix
+import qualified System.Posix.Env.ByteString
+import Test.Tasty
+import Test.Tasty.HUnit
+
+import qualified FileStatus
+import qualified FileStatusByteString
+import qualified Signals001
+
+main :: IO ()
+main = defaultMain $ testGroup "All"
+  [ executeFile001
+  , fileExist01
+  , fileStatus
+  , fileStatusByteString
+  , getEnvironment01
+  , getEnvironment02
+  , getGroupEntry
+  , getUserEntry
+  , processGroup001
+  , processGroup002
+  , queryFdOption01
+  , resourceLimit
```

Comment:
Best to move `resourceLImit` to the bottom of the list, so it does not prematurely interrupt the execution of subsequent tests with a CPU limit exceeded error.